src/main/stackpack/resources/detailed-overview.md

```diff
@@ -9,20 +9,58 @@ The following prerequisites need to be met:
 #### Enabling SAP check
 To enable the SAP check which collects the data from SAP host instance:
 
-Edit the `conf.yaml` file in your agent `conf.d/sap.d` directory, replacing `<sap_host_name>` and `<sap_host_url>` with the unique name of SAP host to identify and URL to connect from your SAP instance.
+Edit the `conf.yaml` file in your agent `/etc/stackstate-agent/conf.d/sap.d` directory, replacing `<sap_host_name>` and `<sap_host_url>` with the unique name of SAP host to identify and URL to connect from your SAP instance.
+
+To Connect to your SAP System, there are 2 ways to do so as explained below:-
+
+##### 1. HTTP Basic Authentication Mechanism
+
+This mechanism allow you to connect just using the `username` and `password` inside the `conf.yaml`.
+
+_As an example, see the below config :-_
+
+```
+# Section used for global SAP check config
+init_config: {}
+
+instances:
+    - host: TEST-01         # <sap_host_name> 
+      url: http://test-01   # <sap_host_url>   
+      user: test            # <username>
+      pass: test            # <password> 
+```
+**NOTE** - Make sure while using this mechanism, you have put `http` in the `url` of the config.
+
+
+##### 2. Client Certificate Authentication Mechanism
+
+This mechanism allow you to connect through the client certificate and private key for the certificate provided. 
```

Comment:
This mechanism allow you to connect using the client certificate and the private key.